### PR TITLE
ENH: Add participant name / ID in plot_alignment

### PR DESCRIPTION
**Pull request by GuillaumeFavelier**
This small PR adds the subject name to the window title.

Closes https://github.com/mne-tools/mne-python/issues/9615

### CODE REVIEW

**Comment by GuillaumeFavelier** [comment]
What do you think @hoechenberger ?

File: mne/viz/_3d.py [hunk 1]
```diff
@@ -891,7 +891,8 @@ def plot_alignment(info=None, trans=None, subject=None, subjects_dir=None,
     del other_bools
 
     # initialize figure
-    renderer = _get_renderer(fig, bgcolor=(0.5, 0.5, 0.5), size=(800, 800))
+    renderer = _get_renderer(fig, name=f'Sensor Alignment: {subject}',
```
Comment (hoechenberger) [inline mne/viz/_3d.py:894]:
```suggestion
    renderer = _get_renderer(fig, name=f'Sensor alignment: {subject}',
```

**Review by hoechenberger — APPROVED** [review]
just a small recommendation, otherwise works like a charm!!!

**Comment by hoechenberger** [comment]
Thank you, @GuillaumeFavelier!!

**Pull request merged**